scaffoldmaker/meshtypes/meshtype_3d_haustra1.py

```diff
@@ -0,0 +1,511 @@
+"""
+Generates a 3-D haustra mesh along a central line, with variable
```

Comment:
I wasn't expecting this to be a separate scaffold, but it could be useful.
Does it make a straight 'colon haustra segment' such as would be repeated on the colon? (See earlier comment).
If so I'd expect the colon to inherit most of its parameters from this scaffold, and use a common function in this scaffold for getting the outer points. Both can use the tube mesh (new version), but this one creates its own single element centre line of the same length as the haustrum.